docs/fundamentals/toc.yml

```
@@ -147,127 +147,125 @@ items:
     href: ../core/whats-new/dotnet-core-2-0.md
   - name: What's new in .NET Standard
     href: ../standard/whats-new/whats-new-in-dotnet-standard.md
-- name: Environment and runtime components
+- name: Tools and productivity
```

I like the idea of moving this up a level. How about:
```suggestion
- name: Tools and diagnostics
```